iothub/service/src/Messaging/Models/Message.cs

```diff
@@ -259,217 +161,81 @@ public string ContentEncoding
         }
 
         /// <summary>
-        /// Gets the dictionary of user properties which are set when user send the data.
-        /// </summary>
-        public IDictionary<string, string> Properties { get; private set; }
-
-        /// <summary>
-        /// Gets the dictionary of system properties which are managed internally.
-        /// </summary>
-        internal IDictionary<string, object> SystemProperties { get; private set; }
-
-        internal Stream BodyStream => _bodyStream;
-
-        internal bool HasBodyStream()
-        {
-            return _bodyStream != null;
-        }
-
-        /// <summary>
-        /// Gets or sets the deliveryTag which is used for server side checkpointing.
-        /// </summary>
-        internal ArraySegment<byte> DeliveryTag { get; set; }
```

Comment:
Did we remove this because it was unused?

Reply:
Did not remove it, just moved it to the bottom of the class.

Reply:
Because internal? Makes sense.

I've tried looking for it, but I can't find it. Line?

Reply:
Never mind. I see it.